### PR TITLE
Offline status

### PR DESCRIPTION
**Pull request by deepaksisodiya**
http://design.scroll.team/stack/editor/error-no-internet.html

### CODE REVIEW

File: src/components/Article.vue [hunk 1]
```diff
@@ -422,8 +438,15 @@ export default {
...
   beforeDestroy() {
     if (this.menuBarTimer) clearInterval(this.menuBarTimer);
+
+    EVENTS.forEach(event =>
+      window.removeEventListener(event, this.updateOnlineStatus)
+    );
   },
   methods: {
+    updateOnlineStatus() {
+      this.isOnline = navigator.onLine;
```
Comment (ujjwalmishra09) [inline src/components/Article.vue:448]: old browsers do not support `nvigator.onLine`. Please write code accordingly.

**Comment by deepaksisodiya** [comment]
discussed about handling this in editor

File: src/components/Article.vue (hunk 1 above)
Comment (deepaksisodiya) [reply src/components/Article.vue:448]: Yes, thanks..adding that condition also,

if browser does not support navigator.onLine then we should consider that online only

Comment (deepaksisodiya) [reply src/components/Article.vue:448]: https://caniuse.com/#search=navigator.onLine

**Pull request merged**